README.md

```diff
@@ -0,0 +1,23 @@
+# NI JavaScript Style Guide
+
+Welcome to NI's internal and external JavaScript and TypeScript linter rules for [ESLint](https://eslint.org/docs/user-guide/getting-started) and [TSLint](https://palantir.github.io/tslint/).
```

Comment:
Could probably leave off saying `internal and external`

Reply:
This was consistent with the Python style guide. However, I happen to agree, so I will remove it.